src/solvers/dg/basis_lobatto_legendre.jl

```diff
@@ -74,6 +74,8 @@ end
 
 @inline nnodes(basis::LobattoLegendreBasis{RealT, NNODES}) where {RealT, NNODES} = NNODES
 
+@inline eachnode(basis::LobattoLegendreBasis) = Base.OneTo(nnodes(basis))
```

Comment:
Maybe you can also replace existing uses of `eachindex(nodes)` with `eachnode(basis)`? I found a few occasions with `grep -r "eachindex.nodes"`.

Reply:
Alright, now there is only one occurrence of `eachindex(nodes)` where `nodes` is passed to a function and `basis` is not defined.